### PR TITLE
Create .znci

### PR DESCRIPTION
**Pull request by cerqiest**
Create .znci file for project tracking

### CODE REVIEW

File: .znci [hunk 1]
```diff
@@ -0,0 +1,7 @@
+NAME="musicbridge"
+DESCRIPTION="MIDI controller for macOS"
+PRIVATE=0
+
+PROJECT_LEAD="gibbiemonster"
+CONTRIBUTORS=["gibbiemonster"]
+INTERNAL_VERSION=1
```
Comment (gibbiemonster) [inline .znci:8]: Version is currently 1.4

**Review by gibbiemonster — CHANGES_REQUESTED** [review]

File: .znci [hunk 2]
```diff
@@ -0,0 +1,8 @@
+NAME="musicbridge"
+DESCRIPTION="MIDI controller for macOS"
```
Comment (gibbiemonster) [inline .znci]: Maybe use the GitHub description?

**Review by gibbiemonster — CHANGES_REQUESTED** [review]

File: .znci (hunk 2 above)
Comment (cerqiest) [reply .znci]: The description here is supposed to be a very short and brief description. Leave it.

Comment (gibbiemonster) [reply .znci]: That's not what this project is

Comment (cerqiest) [reply .znci]: Then can you give me a description?

Comment (gibbiemonster) [reply .znci]: AppleScript/MIDI Bridge for Spotify and Apple Music

**Review by gibbiemonster — APPROVED** [review]
coolio!

**Pull request merged**